lambdas/service/app.py

```diff
@@ -548,7 +549,11 @@ def get_manifest():
     es_td = EsTd()
     # Get the response back
     logger.info("Creating the API response")
-    response = es_td.transform_manifest(filters=filters)
+    try:
+        response = es_td.transform_manifest(filters=filters, format=format)
+    except ValueError as e:
+            return 400, str(e)
```

Comment:
Also, this works? Does it actually return an HTTP response with status 400?

Reply:
I just don't know how that code got there in the first place. I suppose I wrote it, but have no recollection. AFIAK, a hard-coded status code 302 (Chalice `Response`) is returned to that method coming from `ManifestResponse` - I'm not sure how it could be anything else. I removed this code for the time being.